### DISCUSSION TITLE
Please no "social" media. Thanks.

### DISCUSSION THREAD

**Discussion by 772** [community/community#13453] [Other Feature Feedback, Questions, & Ideas]

**Comment by TeoTwawki** [comment]
seconded. Can I just not have a feed? can I turn this off and get a layout without it taking up a gigantic part of my screen? my existing notifications page is the only feed-like thing I need.

If I were the level of social github seems to think I should be, I wouldn't be coding.

**Comment by claywar** [comment]
We all do our thing to contribute to the OSS community in our own ways.  This is my Zen, and I would prefer to not be bombarded by distractions.

**Comment by jfullstackdev** [comment]
there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
